framework/simd-amd.conf

```diff
@@ -1,4 +1,3 @@
-# -*- mode: conf; indent-tabs-mode: t -*-
```

Comment:
Don't remove the mode line. I'm the one who uses them, because my Emacs is configured for spaces by default (which is what we use everywhere, except these Tab-Separated Files and the meson.build files)

Reply:
Personally, I have found https://editorconfig.org/ to be helpful here.

Reply:
Here's my commit message:

> Modelines were placed in some, but not all, source files. Let's remove them for
> now. If there's demand for them, we'll add an EditorConfig file or something at
> the top level.

So, a) mode lines as-needed, or b) scrap modelines in favor of editoconfig?

Reply:
I think we should add an .editorconfig for OpenDCDiag that clarifies that the Meson files use tabs and the C/C++/Perl/Python sources don't.

The simd*conf files should have in-file configuration. Does editorconfig support that?

Reply:
I believe so. You can assign settings based on wildcards.

Reply:
Re-pushed with a new editorconfig-- I can squash it into the "remove modelines" commit once/if folks are fine with it.

Reply:
> I believe so. You can assign settings based on wildcards.

So the answer is "no". I don't want a wildcard. I want something **in** the .conf files because they are used elsewhere.

Reply:
Modelines in the *.conf files is fine. I'll just have the `.editorconfig` not match them. Feels like a happy middle ground.

Reply:
Latest push should do the trick. Resolving convo.